uspace/lib/c/generic/async/server.c

```diff
@@ -190,6 +194,15 @@ typedef struct {
 
 	/** Notification data */
 	void *data;
+
+	/**
+	 * Data of the most recent notification.
+	 * If count > 1, the data for previous notifications are lost.
+	 */
+	ipc_call_t calldata;
+
+	/** How many of this notification arrived since it was last handled. */
```

Comment:
these notifications?

Reply:
I feel like that wouldn't make it clear enough that I'm talking about multiple instances of the same notification (imethod). I'll try rephrasing it.